<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientManagement</class>
 <widget class="QMainWindow" name="ClientManagement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1435</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClientManagement</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit {
    color: black;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>190</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans MT</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Sort by Membership:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="client_search">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>120</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
    background-color: white;
    color: black;
    border: 1px solid gray;
    padding: 5px 10px;
    border-radius: 3px;
  border: 2px solid gray;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="add_client">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>600</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 10pt &quot;MS Shell Dlg 2&quot;;
    border-radius: 50%;
background-color:#586994;
color:white;
 border-width: 0px;
  border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Chart">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>237</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    font: 12pt &quot;MS Shell Dlg 2&quot;;
    background-color: blue; 
    color: white;
    border: none;
    border-radius: 10px;
}
</string>
    </property>
    <property name="text">
     <string>Charts</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>300</y>
      <width>611</width>
      <height>321</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>311</width>
      <height>481</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans MT</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
    border: 2px solid gray;
    border-radius: 5px;
    margin-top: 10px;
background-color:#586994;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center;
    padding: 0 3px;
    font: 63 10pt &quot;Segoe UI Semibold&quot;;
    color :black;
}
QLineEdit{
    background-color: white;
    color: white;
    border: 1px solid gray;
    padding: 5px 10px;
    border-radius: 3px;
  border: 2px solid gray;
}
QPushButton:hover {
    background-color: #45a049;
}
</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
font: 63 12pt &quot;Gill sans MT&quot;;
color :white;
}</string>
     </property>
     <property name="text">
      <string>Client ID</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="client_id">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>224</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    color: black;
}</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
font: 63 12pt &quot;Gill sans MT&quot;;
color :white;
}</string>
     </property>
     <property name="text">
      <string>Client Name:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="client_name">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>150</y>
       <width>224</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    color: black;
}</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="client_pn">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>290</y>
       <width>224</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    color: black;
}</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>260</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
font: 63 12pt &quot;Gill sans MT&quot;;
color :white;
}</string>
     </property>
     <property name="text">
      <string>Phone Number:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>330</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
font: 63 12pt &quot;Gill sans MT&quot;;
color :white;
}</string>
     </property>
     <property name="text">
      <string>Memebership Date:</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="client_date">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>360</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>190</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
font: 63 12pt &quot;Gill sans MT&quot;;
color :white;
}</string>
     </property>
     <property name="text">
      <string>Client Gender:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="client_gd">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>221</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>Male</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Female</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QPushButton" name="PDF">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>237</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill sans MT</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    font: 12pt &quot;Gill sans MT&quot;;
    background-color: blue; 
    color: white;
    border: none; 
    border-radius: 10px;
}
</string>
    </property>
    <property name="text">
     <string>PDF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Search">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>120</y>
      <width>124</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 12pt &quot;MS Shell Dlg 2&quot;;
    border-radius: 50%;
background-color:#586994;
color:white;
 border-width: 0px;
  border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Newest">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>190</y>
      <width>83</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans MT</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Newest</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>124</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill sans MT 13</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 12pt &quot;Gill sans MT&quot; bold;
    border-radius: 50%;
	background-color: rgb(173, 10, 10);
color:#f0f0f0;
 border-width: 0px;
  border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>MENU</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reset_client">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>640</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 10pt &quot;MS Shell Dlg 2&quot;;
    border-radius: 50%;
background-color:#586994;
color:white;
 border-width: 0px;
  border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Oldest">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>190</y>
      <width>83</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans MT</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Oldest</string>
    </property>
   </widget>
   <widget class="QPushButton" name="edit_client">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>600</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 10pt &quot;MS Shell Dlg 2&quot;;
    border-radius: 50%;
background-color:#586994;
color:white;
 border-width: 0px;
  border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>461</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Glacial Indifference</family>
      <pointsize>30</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(61, 119, 139);</string>
    </property>
    <property name="text">
     <string>Client Management</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="client_email">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>540</y>
      <width>224</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    color: black;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>510</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font: 63 12pt &quot;Gill sans MT&quot;;
color :white;
}</string>
    </property>
    <property name="text">
     <string>Email:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Refresh">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>630</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 10pt &quot;MS Shell Dlg 2&quot;;
    border-radius: 50%;
background-color:#586994;
color:white;
 border-width: 0px;
  border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>40</y>
      <width>271</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="chartlayout"/>
   </widget>
   <widget class="QLineEdit" name="arduino">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>410</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
    background-color: white;
    color: black;
    border: 1px solid gray;
    padding: 5px 10px;
    border-radius: 3px;
  border: 2px solid gray;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1435</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
